source/unix/uv_unix_fs.c

```diff
@@ -77,9 +77,11 @@
 
 #define PATH                                                                  \
   do {                                                                        \
-    (req)->path = strdup(path);                                               \
-    if ((req)->path == NULL)                                                  \
-      return -ENOMEM;                                                         \
+    if (path) {                                                               \
```

Comment:
It seems a personal preference, not best practice. :) I am used to webkit source code. Its coding style says your style is wrong (https://webkit.org/code-style-guidelines/#null-false-and-zero). If you have any coding preference that you want to make as rule, please specify explicitly in iotjs coding style guide.